release notes/v0.48.0.md

```diff
@@ -13,7 +13,19 @@ k6 v0.48.0 is here 🎉! This release includes:
 
 ## New features
 
-_optional intro here_
+### Add support for browser module's `page.throttleCPU` [browser#887](https://github.com/grafana/xk6-browser/issues/887)
```

Comment:
I believe we usually link the PR instead of the issue. Should we link https://github.com/grafana/xk6-browser/pull/1095 here?

Reply:
Nice catch, thanks! Fixed in f6774ea2c3cea9b69689c2cd34fab90825e409d7